static/autocomplete.html

```diff
@@ -2,8 +2,8 @@
 <html>
 
 <head>
-    <link rel="stylesheet" href="http://localhost:900/static/awesomplete.css" />
-    <script src="http://localhost:900/static/jquery.min.js"></script>
+    <link rel="stylesheet" href="http://localhost:8080/static/awesomplete.css" />
+    <script src="http://localhost:8080/static/jquery.min.js"></script>
 
```

Comment:
I don't think you want to put in the `host:port` string here.  It's fine for now, but it should be changed to something like:
```html
<link rel="stylesheet" href="/static/awesomplete.css" />
<script src="/static/jquery.min.js"></script>
```
and modify `app.py` to serve static files using something like what's mentioned here:
https://bottlepy.org/docs/0.12/tutorial.html?#routing-static-files

That will allow these files to be served from any host, on any port.